msdbook/tests/test_utils.py

```diff
@@ -0,0 +1,119 @@
+import pytest
```

Comment:
make sure pytest is listed in the requirements, [like on the main branch](https://github.com/IMMM-SFA/msd_uncertainty_ebook/blob/main/pyproject.toml#L66)